scripts/linters/pylint_extensions.py

```diff
@@ -1653,100 +1653,121 @@ class SingleLineCommentChecker(checkers.BaseChecker):
             'Please use capital letter to begin the content of comment.'
         )
     }
-
-    def process_module(self, node):
-        """Process a module to ensure that comments follow correct style.
+    options = ((
+        'allowed-comment-prefixes',
+        {
+            'default': ('int', 'str', 'float', 'bool', 'v'),
```

Comment:
the default should be empty list*

Reply:
If I leave the list empty here, then tests won't pass, as while testing it does not load configuration from pylint. I've tested it by printing the `self.config.allowed_comment_prefix`. It prints an empty list while testing. But if I run linter normally, all configuration was loaded and full list print.

Reply:
set_config do not work with `unittest.TestCase` which we are using to run tests. It is compatible with `testutils.CheckerTestCase`.  I don't think we can use it for now.